applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_strain.cpp

```diff
@@ -55,86 +55,133 @@ LinearPlaneStrain::~LinearPlaneStrain()
 {
 }
 
-void  LinearPlaneStrain::CalculateMaterialResponsePK2 (Parameters& rValues)
+void  LinearPlaneStrain::CalculateMaterialResponsePK2 (Parameters& rParameterValues)
```

Comment:
Ok, I just saw that the name in constitutive_law. h for calculate mnaterial reponse was right:

![imagen](https://user-images.githubusercontent.com/19991680/28660218-711de7da-72b2-11e7-8026-c6a9764bd2a5.png)

I will revert it, but maybe it is a little bit inconsistent